src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Index.cshtml

```diff
@@ -31,52 +33,140 @@
             </tr>
             </thead>
             <tbody>
-            <tr>
-                <td>
-                    <input type="text" class="form-control" asp-for="CustomRedirect.OldUrl">
-                    <span asp-validation-for="CustomRedirect.OldUrl" class="text-danger"></span>
-                </td>
-                <td>
-                    <input type="text" class="form-control" asp-for="CustomRedirect.NewUrl">
-                    <span asp-validation-for="CustomRedirect.NewUrl" class="text-danger"></span>
-                </td>
-                <td class="text-center align-middle">
-                    <input type="checkbox" class="form-check-input" asp-for="CustomRedirect.WildCardSkipAppend">
-                </td>
-                <td>
-                    <select class="form-select" asp-for="CustomRedirect.RedirectType">
-                        <option selected value="@RedirectType.Temporary">@RedirectType.Temporary</option>
-                        <option value="@RedirectType.Permanent">@RedirectType.Permanent</option>
-                    </select>
-                </td>
-                <td>
-                    <div class="d-grid gap-2">
-                        <button type="submit" class="btn btn-primary"
-                                asp-page-handler="create">
-                            <span data-feather="plus"></span> add
-                        </button>
-                    </div>
-                </td>
-            </tr>
-            @foreach (var item in Model.Items)
+            @if (!Model.IsEditing())
```

Comment:
Do we really need to hide it? IMHO removal of the first row causes unwanted flicker.

Reply:
Added modal dialogs.